### ISSUE TITLE
Not compatible with Magento 2.4.0 Commerce

### ISSUE THREAD

**Issue by bernarddelhez** [bitExpert/magento2-force-login#190]
### Preconditions

PHP Version : 7.3.10

Magento Version : 2.4.0

Force Login Module Version : v4.0.2

Third party modules : not relevant for the problem

### Steps to reproduce

1. update the Magento version in the composer.json to `"magento/product-enterprise-edition": "2.4.0"`
2. `composer update`

I think there is the same problem for the Magento 2.4.0 Open Source version

### Expected result

1. no composer dependencies conflicts

### Actual result

```
app@3f7cbd9d8cec:~/html/src$ composer update                                         
Gathering patches for root package.
Loading composer repositories with package information                                                                                                  Updating dependencies (including require-dev)         
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - magento/module-store 101.0.0 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-store 101.0.1 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-store 101.0.2 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-store 101.0.2-p2 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.0 requires php ~5.6.0|7.0.2|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.1 requires php ~5.6.0|7.0.2|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.2 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.3 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.4 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.5 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.6 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.7 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.8 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.0 requires php 7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.1 requires php 7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.9 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.10 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.2 requires php 7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.11 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.3 requires php 7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.12 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.4 requires php 7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.5 requires php ~7.0.13|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.13 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.14 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.6 requires php ~7.0.13|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.15 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.7 requires php ~7.0.13|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 102.0.0 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.16 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 102.0.1 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.8 requires php ~7.0.13|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 102.0.2 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 100.1.17 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.9 requires php ~7.0.13|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.10 requires php ~7.0.13|~7.1.0|~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 102.0.2-p2 requires php ~7.1.3||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - magento/module-customer 101.0.11 requires php ~7.0.13||~7.1.0||~7.2.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 3.2.0 requires magento/module-customer ~100.1|~101.0|~102.0 -> satisfiable by magento/module-customer[102.0.5, 102.0.5-p2, 102.0.4-p2, 102.0.4-p1, 102.0.3-p1, 102.0.4, 101.0.11, 102.0.2-p2, 101.0.10, 102.0.3, 101.0.9, 100.1.17, 102.0.2, 101.0.8, 102.0.1, 100.1.16, 102.0.0, 101.0.7, 100.1.15, 101.0.6, 100.1.14, 100.1.13, 101.0.5, 101.0.4, 100.1.12, 101.0.3, 100.1.11, 101.0.2, 100.1.10, 100.1.9, 101.0.1, 101.0.0, 100.1.8, 100.1.7, 100.1.6, 100.1.5, 100.1.4, 100.1.3, 100.1.2, 100.1.1, 100.1.0].
    - bitexpert/magento2-force-customer-login 3.2.0 requires magento/module-customer ~100.1|~101.0|~102.0 -> satisfiable by magento/module-customer[102.0.5, 102.0.5-p2, 102.0.4-p2, 102.0.4-p1, 102.0.3-p1, 102.0.4, 101.0.11, 102.0.2-p2, 101.0.10, 102.0.3, 101.0.9, 100.1.17, 102.0.2, 101.0.8, 102.0.1, 100.1.16, 102.0.0, 101.0.7, 100.1.15, 101.0.6, 100.1.14, 100.1.13, 101.0.5, 101.0.4, 100.1.12, 101.0.3, 100.1.11, 101.0.2, 100.1.10, 100.1.9, 101.0.1, 101.0.0, 100.1.8, 100.1.7, 100.1.6, 100.1.5, 100.1.4, 100.1.3, 100.1.2, 100.1.1, 100.1.0].
    - bitexpert/magento2-force-customer-login 3.2.0 requires magento/module-customer ~100.1|~101.0|~102.0 -> satisfiable by magento/module-customer[102.0.5, 102.0.5-p2, 102.0.4-p2, 102.0.4-p1, 102.0.3-p1, 102.0.4, 101.0.11, 102.0.2-p2, 101.0.10, 102.0.3, 101.0.9, 100.1.17, 102.0.2, 101.0.8, 102.0.1, 100.1.16, 102.0.0, 101.0.7, 100.1.15, 101.0.6, 100.1.14, 100.1.13, 101.0.5, 101.0.4, 100.1.12, 101.0.3, 100.1.11, 101.0.2, 100.1.10, 100.1.9, 101.0.1, 101.0.0, 100.1.8, 100.1.7, 100.1.6, 100.1.5, 100.1.4, 100.1.3, 100.1.2, 100.1.1, 100.1.0].
    - bitexpert/magento2-force-customer-login 4.0.0 requires magento/module-store ~101.0.0|~101.0.1|~101.0.3 -> satisfiable by magento/module-store[101.0.5, 101.0.4-p2, 101.0.4-p1, 101.0.3-p1, 101.0.4, 101.0.2-p2, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 4.0.2 requires magento/module-store ~101.0.0|~101.0.1|~101.0.3 -> satisfiable by magento/module-store[101.0.5, 101.0.4-p2, 101.0.4-p1, 101.0.3-p1, 101.0.4, 101.0.2-p2, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 4.0.0 requires magento/module-store ~101.0.0|~101.0.1|~101.0.3 -> satisfiable by magento/module-store[101.0.5, 101.0.4-p2, 101.0.4-p1, 101.0.3-p1, 101.0.4, 101.0.2-p2, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 4.0.2 requires magento/module-store ~101.0.0|~101.0.1|~101.0.3 -> satisfiable by magento/module-store[101.0.5, 101.0.4-p2, 101.0.4-p1, 101.0.3-p1, 101.0.4, 101.0.2-p2, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - Can only install one of: magento/module-customer[102.0.5-p2, 103.0.0].
    - Can only install one of: magento/module-customer[102.0.4-p2, 103.0.0].
    - Can only install one of: magento/module-customer[102.0.5, 103.0.0].
    - Can only install one of: magento/module-customer[102.0.4-p1, 103.0.0].
    - Can only install one of: magento/module-customer[102.0.3-p1, 103.0.0].
    - Can only install one of: magento/module-customer[102.0.4, 103.0.0].
    - Can only install one of: magento/module-customer[102.0.3, 103.0.0].
    - Can only install one of: magento/module-customer[103.0.0, 102.0.5].
    - Can only install one of: magento/module-store[101.0.4-p2, 101.1.0].
    - Can only install one of: magento/module-store[101.0.5, 101.1.0].
    - Can only install one of: magento/module-store[101.0.4-p1, 101.1.0].
    - Can only install one of: magento/module-store[101.0.3-p1, 101.1.0].
    - Can only install one of: magento/module-store[101.0.4, 101.1.0].
    - Can only install one of: magento/module-store[101.0.3, 101.1.0].
    - Can only install one of: magento/module-store[101.1.0, 101.0.5].
    - magento/module-website-restriction 100.4.0 requires magento/module-customer 103.0.* -> satisfiable by magento/module-customer[103.0.0].
    - magento/module-website-restriction 100.4.0 requires magento/module-store 101.1.* -> satisfiable by magento/module-store[101.1.0].
    - magento/product-enterprise-edition 2.4.0 requires magento/module-website-restriction 100.4.0 -> satisfiable by magento/module-website-restriction[100.4.0].
    - bitexpert/magento2-force-customer-login 2.2.0 requires magento/module-customer ~101.0 -> satisfiable by magento/module-customer[101.0.11, 101.0.10, 101.0.9, 101.0.8, 101.0.7, 101.0.6, 101.0.5, 101.0.4, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 2.3.0 requires magento/module-customer ~101.0 -> satisfiable by magento/module-customer[101.0.11, 101.0.10, 101.0.9, 101.0.8, 101.0.7, 101.0.6, 101.0.5, 101.0.4, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login v3.0.0 requires magento/module-customer ~101.0 -> satisfiable by magento/module-customer[101.0.11, 101.0.10, 101.0.9, 101.0.8, 101.0.7, 101.0.6, 101.0.5, 101.0.4, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 3.0.0 requires magento/module-customer ~101.0 -> satisfiable by magento/module-customer[101.0.11, 101.0.10, 101.0.9, 101.0.8, 101.0.7, 101.0.6, 101.0.5, 101.0.4, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 2.2.0 requires magento/module-customer ~101.0 -> satisfiable by magento/module-customer[101.0.11, 101.0.10, 101.0.9, 101.0.8, 101.0.7, 101.0.6, 101.0.5, 101.0.4, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 2.3.0 requires magento/module-customer ~101.0 -> satisfiable by magento/module-customer[101.0.11, 101.0.10, 101.0.9, 101.0.8, 101.0.7, 101.0.6, 101.0.5, 101.0.4, 101.0.3, 101.0.2, 101.0.1, 101.0.0].
    - bitexpert/magento2-force-customer-login 3.1.0 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 3.0.1 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.1.0 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.0.2 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.0.1 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.0.0 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.3.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.3.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.4 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.3 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.2 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.7 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.6 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.5 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.4 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.3 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.2 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.0.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.0.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 3.1.0 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 3.0.1 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6|~7.1.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.1.0 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.0.2 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.0.1 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 2.0.0 requires php ~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.3.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.3.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.4 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.3 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.2 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.2.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.7 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.6 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.5 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.4 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.3 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.2 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.1.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.0.1 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - bitexpert/magento2-force-customer-login 1.0.0 requires php ~5.5.0|~5.6.0|~7.0.0 -> your PHP version (7.3.10) does not satisfy that requirement.
    - Installation request for magento/product-enterprise-edition 2.4.0 -> satisfiable by magento/product-enterprise-edition[2.4.0].
    - Installation request for bitexpert/magento2-force-customer-login * -> satisfiable by bitexpert/magento2-force-customer-login[1.0.0, 1.0.1, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.1.6, 1.1.7, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.3.0, 1.3.1, 2.0.0, 2.0.1, 2.0.2, 2.1.0, 2.2.0, 2.3.0, 3.0.1, 3.1.0, 3.2.0, 4.0.0, 4.0.2, 3.0.0].
```

**Comment by shochdoerfer** [comment]
Hey @bernarddelhez thanks for raising the issue. And thanks for your first issue in this repo :) 

Mind providing a PR to fix it? Would be fine for me if the next major release would only be compatible with Magento 2.4. That would reduce maintenance effort.

**Comment by shochdoerfer** [comment]
You are right both Magento versions are affected as we did not find the time to upgrade the module to work with 2.4. Might "just" be the Composer issue, could potentially be more to fix.

**Comment by bernarddelhez** [comment]
@shochdoerfer here is the PR https://github.com/bitExpert/magento2-force-login/pull/191

**Comment by dmcmillin** [comment]
even if you change the dependency issue via composer, on 2.4.0 it says too many redirects. Just a heads up.

**Comment by arnoudhgz** [comment]
@bernarddelhez did you check the comment above against your PR? Did you have the redirect issues that @dmcmillin mentioned?

**Comment by bernarddelhez** [comment]
@arnoudhgz I also had the problem with "too many redirects" and I changed the following config
`bin/magento config:set customer/startup/redirect_dashboard 0`
This configuration change works on a Magento Commerce 2.4.0 live system.
I did not look deeper in the code to find the reason but it works.

**Comment by websharp** [comment]
@bernarddelhezI we added some fixes to the master, do you mind testing your issue, if everything works fine now or report any new issue occured?

**Comment by shochdoerfer** [comment]
The dev-master version of this module has been made compatible with 2.4.0 - 2.4.4, we'll release a new version in the next few days.

I could not reproduce the "too many redirects" with the dev-master version and Magento 2.4.4. This might have been fixed with #206 or #209.
